source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/skin.html

```
@@ -1,63 +1,75 @@
+        <li><hr/></li>
+        <li class='caMenuItem debugging'><?tr("Debugging");?></li>
+      </ul>
+    </div>
+  </div>
+  <div class='mainArea'>
+    <div class='searchArea onlyShowWithFeed'>
+      <div class='menuItemsHidden'>
+        <div class='showMenuButton'></div>
+      </div>
+      <span id='searchFilter'>
+        <input type='text' id='searchBox' placeholder='<?tr('Search');?>'>
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)